event-handlers/src/main/java/no/sikt/nva/nvi/events/evaluator/EvaluatorService.java

```diff
@@ -129,6 +146,22 @@ private static PublicationDate mapToPublicationDate(JsonNode publicationDateNode
                    .orElse(new PublicationDate(null, null, year.textValue()));
     }
 
+    private boolean isPublishedBeforeOrSameAsLatestClosedPeriod(PublicationDate publicationDate) {
```

Comment:
This method checks only if publicationYear is before latestClosedPeriod, isn't it? 
Could be renamed to something like `isWithinClosedPeriod`, checks if publication date falls within a closed period.